controllers/perconaservermysqlbackup_controller.go

```diff
@@ -315,3 +332,197 @@ func (r *PerconaServerMySQLBackupReconciler) getBackupSource(ctx context.Context
 
 	return src, nil
 }
+
+const (
+	finalizerDeleteBackup = "delete-backup"
```

Comment:
You can remove this const and just use the string directly since it is used only in one place.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/158/commits/4528a32e0c26a104b9842b349e6439428d2c7bb7